ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-commands.sh

```diff
@@ -27,6 +27,16 @@ CONFIG="${SHARED_DIR}/install-config.yaml"
 base_domain=$(<"${SHARED_DIR}"/basedomain.txt)
 machine_cidr=$(<"${SHARED_DIR}"/machinecidr.txt)
 
+RESOURCE_POOL_DEF=""
+VERSION=$(echo "${JOB_NAME}" | grep -o -E '4\.[0-9]+')
+if [ ! -z ${VERSION} ]; then
+    Z_VERSION=$(echo ${VERSION} | cut -d'.' -f2)
+    if [ ${Z_VERSION} -gt 9 ]; then
+        echo "4.x installation is later than 4.9, will install with resource pool"
+        RESOURCE_POOL_DEF="ipi-ci-clusters"
```

Comment:
I think this needs to be `RESOURCE_POOL_DEF="resourcePool: ipi-ci-clusters"`

Reply:
yep, good catch!